cloudferry_devlab/cloudferry_devlab/tests/base.py

```diff
@@ -83,6 +84,11 @@ def __init__(self, config, configuration_ini, cloud_prefix='SRC',
         self.server_groups_supported = self.openstack_release in ['icehouse',
                                                                   'juno']
 
+    def mysql_connector(self, db_name, position):
+        config = self.configuration_ini["%s_mysql" %
+                                        (position or self.cloud_prefix)]
```

Comment:
if `position` is not an optional argument you don't need `position or self.cloud_prefix`, just `position`